corehq/apps/dump_reload/sql/dump.py

```diff
@@ -19,7 +19,8 @@
 from corehq.sql_db.config import plproxy_config
 
 # order is important here for foreign key constraints
```

Comment:
Know if this is still true?  Looks like the order is controlled here, at any rate:
https://github.com/dimagi/commcare-hq/blob/60db68bbbe100a0aecd9b8447c3ad078bc10a77e/corehq/apps/dump_reload/sql/dump.py#L180-L180

I think `defaultdict` is reliably ordered as of Python 3.6 or 3.7, so it may not matter.

Reply:
i suspect that's not true since as you point out the order is determined outside the dict.